donkeycar/parts/actuator.py

```diff
@@ -166,6 +166,60 @@ def set_pulse(self, pulse):
     def run(self, pulse):
         self.set_pulse(pulse)
 
+class VESC:
+    ''' 
+    VESC Motor controler using pyvesc
```

Comment:
Thanks for the great header.

Reply:
No problem! It was written by @ivanferrier